src/data_format/mod.rs

```diff
@@ -4,92 +4,55 @@
 //! In order to support multiple format versions simultaneously, this module wraps around different
 //! serialization/deserialization implementations and can automatically dispatch to the appropriate
 //! one.
-
-mod storage;
-
-pub(crate) mod utils;
-
-use crate::cosmetic_filter_cache::CosmeticFilterCache;
-use crate::flatbuffers::unsafe_tools::VerifiedFlatbufferMemory;
-use crate::network_filter_list::NetworkFilterListParsingError;
+//!
+//! The current .dat file format:
+//! 1. magic (4 bytes)
+//! 2. seahash of the data (8 bytes)
+//! 3. data (the rest of the file)
```

Comment:
we still need the version in the header, in the case of the hash starting with a `1` byte

Reply:
returned `version` to .dat header.